api/src/test/java/org/openmrs/api/ObsServiceTest.java

```diff
@@ -1062,7 +1062,7 @@ public void getObservationCount_shouldReturnCountOfObsWhosePersonIsAPatientOnly(
 		Integer count = obsService.getObservationCount(null, null, null, null,
 		    Collections.singletonList(PERSON_TYPE.PATIENT), null, null, null, null, false, null);
 		
-		Assert.assertEquals(15, count.intValue());
```

Comment:
Why increase the count to 16?

Reply:
On making the changes the number of persons that are patients increased to 16.

Reply:
This increased after making person_id=7 from person_id=9 in obs_id=9. This is in accordance with standardTestDataset